### ISSUE TITLE
Setup deployment pipeline/branches

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#30]
## Acceptance criteria
* Two different branches:
  - One for stable (production) releases
  - One for unstable (testing) releases
 * Restriction on commiting to these branches
 * Automatic deployment on push

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
